app/views/components/blockEditor.html

```diff
@@ -19,9 +19,9 @@
     <button class="btn btn-danger" ng-click="deleteBlock(block.id, true)" ng-disabled="!canDelete"><i class="fa fa-trash-o"></i> <translate>Delete</translate></button>
     <span class="btn btn-default" ui-tree-handle><i class="fa fa-bars"></i> <translate>Move</translate></span>
   </div>
-  <label>{{block.title}} <em class="required" ng-show="block.required">*</em></label>
+  <label ng-bind-html="block.title" tooltip="{{'Supports hyperlink html format.' | translate}}"><em class="required" ng-show="block.required">*</em></label>
```

Comment:
I think these would be better next to the word "Title" in the edit panel. I think this hint is more relevant to a user typing a title than hovering over the question title.

I think it would be good to add the font awesome question mark and have the popover or tooltip appear when hovering over that. (I don't care whether you use a popover or tooltip. Other parts of the site seem to be using popovers as tooltips) There's a bunch of these question marks on different sections of the event details page
<img width="135" alt="screen shot 2016-10-04 at 11 31 59 am" src="https://cloud.githubusercontent.com/assets/756501/19087246/31485c64-8a26-11e6-86ba-2c3aeac1a50d.png">

I think the wording should be "Supports HTML links" and should include an example.
```
Supports HTML links
Example: <a href="https://www.google.com">Link to Google</a>
```

You can encode that as `&lt;a href=&quot;https://www.google.com&quot;&gt;Link to Google&lt;/a&gt;` but getting a new line there and not having the example link wrap might not be straightforward.

Do you feel like that is doable?